CONTRIBUTING.md

```diff
@@ -8,6 +8,32 @@ We're excited you're interested in contributing to Marquez! We'd love your help,
 
 We feel that a welcoming community is important and we ask that you follow the [Contributor Covenant Code of Conduct](https://github.com/MarquezProject/marquez-airflow/blob/main/CODE_OF_CONDUCT.md) in all interactions with the community.
 
+# Development
+We recommend installing the following tools to run the tests:
+
+Install pyenv: `brew install pyenv pyenv-virtualenv`
+Add it to your profile: `~/.zshrc`

```

Comment:
We may want to mention `~/.bashrc` as well

Reply:
done